src/pages/Contact.js

```diff
@@ -1,7 +1,176 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { TextField, Button, Stack, Box, Modal, Paper } from "@mui/material";
+import { useForm, Controller } from "react-hook-form";
+import { DevTool } from "@hookform/devtools";
+import axios from "axios";
 
 function Contact() {
-  return <div>Contact</div>;
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+    reset,
+  } = useForm({
+    defaultValues: {
+      firstName: "",
+      lastName: "",
+      email: "",
+      orderInfo: "",
+    },
+  });
+  const [dataFromServer, setDataFromServer] = useState("");
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  useEffect(() => {
+    const fetDataFromServer = async () => {
```

Comment:
Better use pattern for use Effect is to have fetDataFromServer outside the effect and then call within the useeffect

also weird spelling - do you want it to fetchDataFromServer - just fetechData, or getData